src/assets/scripts/client/airline/flightNumberBuilder.js

```diff
@@ -0,0 +1,51 @@
+import { choose } from '../utilities/generalUtilities';
+
+const ALPHA = 'abcdefghijklmnopqrstuvwxyz';
+const NUMERIC = '0123456789';
+
+/**
+ * This picks a random number. If it is the first value within the callsign (ie. i === 0), then it picks a
+ * number between 1 and 9. Otherwise, it picks a number between 0 and 9.
+*/
+const generateNumericValue = (i) => {
+    if (i === 0) {
+        return choose(NUMERIC.substr(1));
+    }
+
+    return choose(NUMERIC);
+};
+
+/**
+ * Self-explanatory; this function picks a random letter and returns it.
+*/
+const generateAlphaValue = () => {
```

Comment:
this should be a private function: `_generateAlphaValue` and please remove the _self-explanatory_ part of the docblock.

be sure to also add:
```
@function
@param
@return
```
and applicable data types to the docblock